### PR TITLE
Add support for "live" navigation from a liveview

### PR DESCRIPTION
**Pull request by floodfx**

### CODE REVIEW

File: live/liveview.go [hunk 1]
```diff
@@ -232,6 +232,7 @@ func (x *WebsocketHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		msg:           make(chan *phx.Msg),       // TODO: buffered?
 		info:          make(chan *Info),          // TODO: buffered?
 		upload:        make(chan *phx.UploadMsg), // TODO: buffered?
+		nav:           make(chan *phx.Nav),       // TODO: buffered?
```
Comment (kevboh) [inline live/liveview.go:235]: laughing at these TODOs

File: live/liveview.go [hunk 2]
```diff
@@ -778,6 +783,47 @@ func PageTitle(ctx context.Context, newTitle string) {
...
+		kind = "replace"
+	}
+
+	// call HandleParams if view implements ParamsHandler
+	hp, ok := s.view.(ParamsHandler)
+	if ok {
+		err := hp.HandleParams(ctx, to)
+		if err != nil {
+			// TODO better error handling
+			log.Println(err)
```
Comment (kevboh) [inline live/liveview.go:814]: Can we return the error and let the framework user handle?

**Review by kevboh — APPROVED** [review]
lgtm. see second comment for a suggestion

File: live/liveview.go (hunk 2 above)
Comment (floodfx) [reply live/liveview.go:814]: good call

File: live/liveview.go (hunk 1 above)
Comment (floodfx) [reply live/liveview.go:235]: removing

**Pull request merged**